runtime/worker_bootstrap.rs

```diff
@@ -23,6 +27,31 @@ pub struct BootstrapOptions {
   pub user_agent: String,
 }
 
+impl Default for BootstrapOptions {
+  fn default() -> Self {
+    let cpu_count = thread::available_parallelism()
+        .map(|p| p.get())
+        .unwrap_or(1);
+
+    let runtime_version = env!("CARGO_PKG_VERSION").into();
```

Comment:
Having the runtime version the version of the `deno_runtime` crate is probably a better idea than whatever the consumer can specify here.